networkit/_NetworKit.pyx

```diff
@@ -8666,14 +8666,26 @@ cdef class ApproxCloseness(Centrality):
 		return (<_ApproxCloseness*>(self._this)).getSquareErrorEstimates()
 
 
+cdef extern from "<networkit/centrality/PageRank.hpp>" namespace "NetworKit::PageRank":
+
+	cdef enum Norm:
+		L1Norm = 0
+		L2Norm = 1
+
+class _PageRankNorm(object):
```

Comment:
Why is this additional `_PageRankNorm`-class introduced here?

Reply:
I need that in `centrality.py` so one can set the norm of `PageRank` to `networkit.centrality.Norm.L1Norm/L2Norm`.

Reply:
Ah I see ... thought this could be done by accessing the public enums from the C++ code.